extensions/resource-deployment/package.json

```diff
@@ -1,368 +1,370 @@
+								"displayName": "%bdc-deployment-target-new-aks%"
+							},
+							{
+								"name": "existing-aks",
+								"displayName": "%bdc-deployment-target-existing-aks%"
+							},
+							{
+								"name": "existing-kubeadm",
+								"displayName": "%bdc-deployment-target-existing-kubeadm%"
+							}
+						]
+					}
+				],
+				"providers": [
+					{
+						"wizard": {
+							"type": "new-aks",
+							"notebook": "%bdc-2019-aks-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azure-cli"
+							},
+							{
+								"name": "azdata",
+								"version": "15.0.2070"
+							}
+						],
+						"when": "target=new-aks&&version=bdc2019"
+					},
+					{
+						"wizard": {
+							"type": "existing-aks",
+							"notebook": "%bdc-2019-existing-aks-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azdata"
+							}
+						],
+						"when": "target=existing-aks&&version=bdc2019"
+					},
+					{
+						"wizard": {
+							"type": "existing-kubeadm",
+							"notebook": "%bdc-2019-existing-kubeadm-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azdata"
+							}
+						],
+						"when": "target=existing-kubeadm&&version=bdc2019"
+					}
+				],
+				"agreement": {
+					"template": "%bdc-agreement%",
+					"links": [
+						{
+							"text": "%bdc-agreement-privacy-statement%",
+							"url": "https://go.microsoft.com/fwlink/?LinkId=853010"
+						},
+						{
+							"text": "%bdc-agreement-bdc-eula%",
+							"url": "https://go.microsoft.com/fwlink/?LinkId=2002534"
+						},
+						{
+							"text": "%bdc-agreement-azdata-eula%",
+							"url": "https://aka.ms/azdata-eula"
+						}
+					]
+				}
+			},
+			{
+				"name": "sql-windows-setup",
+				"displayIndex": 1,
+				"displayName": "%resource-type-sql-windows-setup-display-name%",
+				"description": "%resource-type-sql-windows-setup-description%",
+				"platforms": [
+					"win32"
+				],
+				"icon": {
+					"light": "./images/sql_server_on_windows.svg",
+					"dark": "./images/sql_server_on_windows_inverse.svg"
+				},
+				"options": [
+					{
+						"name": "version",
+						"displayName": "%version-display-name%",
+						"values": [
+							{
+								"name": "sql2017",
+								"displayName": "%sql-2017-display-name%"
+							},
+							{
+								"name": "sql2019",
+								"displayName": "%sql-2019-display-name%"
+							}
+						]
+					}
+				],
+				"providers": [
+					{
+						"downloadUrl": "https://go.microsoft.com/fwlink/?linkid=853016",
+						"requiredTools": [],
+						"when": "version=sql2017"
+					},
+					{
+						"webPageUrl": "https://www.microsoft.com/evalcenter/evaluate-sql-server-2019-rc",
+						"requiredTools": [],
+						"when": "version=sql2019"
+					}
+				]
+			}
+		]
+	},
+	"dependencies": {
+		"getos": "^3.1.1",
```

Comment:
Out of curiosity - what made you chose this package? Did you look at any others to see if there's any other options? Did you look at the package to make sure it's not doing anything weird? (for security/perf reasons)

Reply:
I chose this one based on popularity and usage by other products. 

This is one is the most functional and popular for this use. It itself does not depend on any uncommon dependency that we will have to pull in. Some of the others out there extend is functionality but depend on it. It has no outstanding security issues at least as reported by the automated git checks and the implementation was simple but extensible (so newer platforms discovery is easily added) using appropriate os commands. 

Two other candidates that we can use are linux-os-info and linux-release-info. They are both very similar, they do not provide as much coverage and have smaller codebase behind it. So if we want to go for smaller codebase based dependency then I can switch to linux-os-info.

Reply:
I might have to switch just to solve the linux packaging issue as well.

Reply:
Why would you need to switch? You should just need to add it as an external to the config similar to https://github.com/microsoft/azuredatastudio/blob/master/extensions/mssql/extension.webpack.config.js#L18

Reply:
It was might, I have switched because one I am using now is lighter weight and supports both sync/async and Promise versions out of the box. Check the implementation now to see what I mean.

Reply:
> Why would you need to switch? You should just need to add it as an external to the config similar to https://github.com/microsoft/azuredatastudio/blob/master/extensions/mssql/extension.webpack.config.js#L18

Good to know about this. I did not know this before.